File: download.js

// Aqua Tools
// download.js

const https = require('https');
const fs = require('fs');

var download = function (uri, filename) {
    return new Promise(function (resolve, reject) {
        return https
            .request(uri, function (res) {
                res
                    .pipe(fs.createWriteStream(filename))
                    .on("close", resolve)
                    .on("error", reject);
            })
            .end();
    });
};

const arg = process.argv;

download(arg[2], arg[3]);

File: download.py

# Aqua Tools
# download.py

import requests
import sys

arg = sys.argv

url = arg[1]
filename = arg[2]

urlData = requests.get(url).content

with open(filename, mode='wb') as f:  # wb でバイト型を書き込める
    f.write(urlData)
